XCUITests/DataManagementTests.swift

```diff
@@ -6,28 +6,11 @@ import XCTest
 
 class DataManagementTests: BaseTestCase {
 
-//    func testWebSiteDataEnterFirstTime() {
-//        navigator.goto(URLBarOpen)
-//        navigator.back()
-//        waitForExistence(app.buttons["TabToolbar.menuButton"], timeout: 5)
-//        navigator.performAction(Action.AcceptClearAllWebsiteData)
-//        let expectedWebsiteDataEntries2 = app.tables.cells.count
-//        XCTAssertEqual(expectedWebsiteDataEntries2, 1)
-//        navigator.openURL("example.com")
-//        navigator.goto(WebsiteDataSettings)
-//        let expectedWebsiteDataEntries3 = app.tables.cells.count
-//        XCTAssertEqual(expectedWebsiteDataEntries3, 2)
-//    }
-
-    /* Disabled failing on BR
-    // Testing the search bar, search results and 'Show More' option.
+    // Testing the search bar, and clear website data option
     func testWebSiteDataOptions() {
-        // Visiting some websites to create Website Data needed to reveal the "Show More" button
```

Comment:
Modify to check the default options as generating data in this folder is pretty difficult/unpredictable